test/issue1505-single-file-package-dynamic-library.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+cd ${CURR_DIR}
+rm -f libsingle-file-test-dynamic-library.so
+rm -f single-file-test-dynamic-library.dll
+
+${DUB} build --single issue1505-single-file-package-dynamic-library.d
+if [ ! -f libsingle-file-test-dynamic-library.so ] && [ ! -f single-file-test-dynamic-library.dll ]; then
```

Comment:
```suggestion
if [ ! -f libsingle-file-test-dynamic-library.{dll,so,dylib} ]; then
```

Reply:
The windows `dll` and the linux `so` have different names. The linux so starts with `lib`.

Yes, I was also wondering why the MacOS test pipeline were succeeding although the `dylib` was missing.

Reply:
> The linux so starts with lib

Ah yes, then this should do it:

```
{lib,}single-file-test-dynamic-library.{dll,so,dylib}
```